### DISCUSSION TITLE
Are the full, per-respondant answers available anywhere?

### DISCUSSION THREAD

**Discussion by peter-mlabs** [cardano-foundation/state-of-the-developer-ecosystem#2] [F.A.Q.] [answered]
The [available data](https://github.com/cardano-foundation/state-of-the-developer-ecosystem/blob/main/2023/data/answers.json) appears to be collected in a list, which means we can't determine correlations between questions. 

For instance, we can't determine whether "years of experience in writing and developing software" (Q1) is correlated with preferred language for writing plutus (Q11). Suppose that all of your 3+ years developers are switching to aiken over haskell; that's a very different picture than if many of the aiken developers have <3 years experience, or if there's no correlation at all. It

[EDIT] It seems that some filters are available. But is the full data available for inspection?

**Comment by KtorZ** [comment, marked as answer]
Answered here -> https://github.com/input-output-hk/Developer-Experience-working-group/issues/76#issuecomment-1794238122
